.github/workflows/basic.yaml

```diff
@@ -12,7 +12,7 @@ name: Basic
 jobs:
   test:
     name: Test Suite
-    runs-on: ubuntu-latest-large
+    runs-on: ubuntu-8-core-32-gb
```

Comment:
Why did you have to change the system here?

Reply:
it's much cheaper 😂